scripts/bert/embedding.py

```diff
@@ -194,7 +222,15 @@ def oov(self, batches, oov_way='avg'):
                         help='file for encoding')
 
     args = parser.parse_args()
-    context = mx.gpu(args.gpu) if args.gpu else mx.cpu()
+
+    level = logging.DEBUG if args.verbose else logging.INFO
```

Comment:
`verbose` argument is not in the parser.

Reply:
Nice catch, I missed it while merging a rebase, will add it.